src/login/LoginViewController.js

```diff
@@ -251,11 +251,9 @@ export class LoginViewController implements ILoginViewController {
 			})
 			.then(() => logins.loginComplete()).then(() => {
 			// don't wait for it, just invoke
-			if (isApp()) {
-				import("../native/common/FileApp")
-					.then(({fileApp}) => fileApp.clearFileData())
-					.catch((e) => console.log("Failed to clean file data", e))
-			}
+			import("../native/common/FileApp")
```

Comment:
nope, don't always load fileApp, we don't want to import it on web

Reply:
oh yeah...